### ISSUE TITLE
Experience Showing Up

### ISSUE THREAD

**Issue by JZL** [mrzool/cv-boilerplate#4]
Sorry if I'm missing something obvious but I can't seem to get the "experience" section to show up. I cloned the repo and ran `make` again but it doesn't have it included. If I delete the if statement in the template.tex file, it works.

Thanks!

**Comment by JZL** [comment]
From what I can see from the given templates ( ie https://github.com/jgm/pandoc-templates/blob/master/default.latex line 39), the way pandoc does it is have two seperate variables i.e. an $if(experience)$ for whether or not to include the main heading and then a $for(experienceoptions)$ for the rest. I agree this is not ideal but I am curious why the if statement for the multi-variable block doesn't work

**Comment by mrzool** [comment]
Hello and sorry for the delay. I'm not following: in my template, the experience section is not inside an if statement, so I'm not sure which if statement you deleted. Also, I'm not using the pandoc template you're linking. The only template used in this repo is the one present in this repo, `template.tex`.

The only thing that comes to my mind that could cause such an issue is something wrong in your `details.yml`. Try to maintain the structure of the blocks, indentation and all. This is how you need to structure your content:

``` yaml
experience:
- years: 1879--1889
  employer: Freiberufler
  job: Freier Philisoph
  city: Sils-Maria
```

Make sure you don't have any syntax error or anything that could confuse the parser. [This page](https://learnxinyminutes.com/docs/yaml/) is a good reference for the YAML syntax.

If you still have issues, please post the content of your `details.yml` here.

**Comment by JZL** [comment]
I apologize, I meant the education section which does have a [if statement in template.tex](https://github.com/mrzool/cv-boilerplate/blob/master/template.tex#L129)

``` tex
$if(education)$
```

 and is not working with the included `details.yml` (I also passed it through a [yaml linter](http://www.yamllint.com/) to be sure)

**Comment by mrzool** [comment]
I cannot reproduce. `make` works fine on my computer with the provided files.

Can you confirm that the education section doesn't show up in your output after running `make` with a fresh clone? Didn't you change any setting whatsoever? Also, please double-check if It's not getting pushed down to a second page by preceding content.

**Comment by JZL** [comment]
I `git clone`'d your repo directly. The only change I made was to change the font to "Cardo" because the "Hoefler Text" font is hard to get on Linux. The pdf outputted is available [here](https://dl.dropboxusercontent.com/u/24397004/output.pdf).

This is not a big deal, I can just fork yours or remember to delete the `if statement` whenever I use it. I thought I installed all the dependencies in a standard way but I might not have. Once I delete that it works perfectly and, as I said, I really enjoy these boilerplate repos :+1:
